finesse/gui/temp_control.py

```diff
@@ -161,6 +173,25 @@ def __init__(self, num_channels: int) -> None:
         layout = self._create_controls()
         self.setLayout(layout)
 
+        self._begin_polling()
```

Comment:
As with the EM27 scraper, it might be worth polling the device once before starting the timer so the GUI is populated with values as soon as it opens.